packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -451,6 +461,23 @@ export default decorate([
               </div>
             </fieldset>
           )}
+          {type !== 's3' && (
+            <fieldset className='form-group form-group'>
+              <div className='input-group form-group'>
+                <span className='align-middle'>
+                  {_('remoteUseVhdDirectory')}
+                  <Tooltip content={_('remoteUseVhdDirectoryTooltip')}>
+                    <Icon icon='info' size='lg' />
+                  </Tooltip>
+                </span>
+                <Toggle
+                  className='align-middle pull-right'
+                  onChange={effects.setUseVhdDirectory}
```

Comment:
`effects.linkState` doesn't work, here?

Reply:
General example, not sure of this specific code:

```js
<input name='foo' onChange={effects.linkState} value={state.foo} />
```

Reply:
So do you want to change this? Does it work? I see that we didn't use it for the other Toggle.

Reply:
* I don't really care, you are both the experts in `reaclette`, not me . As long as it works it's ok for me.
* since the backend PR is merged, I did a full test and have been able to edit and create a remote with and without the optio
* it is litterally used one line above in the effects for setAllowUnauthorized